### PR TITLE
feat: updated genesis for v0.3.0

### PR DESCRIPTION
**Pull request by twoshark**

### CODE REVIEW

File: delorean-1/genesis.json [hunk 1]
```diff
@@ -138,20 +138,26 @@
...
-        "finality_delay": "0",
-        "registry_coordinator_address": "0x55d1E957a0071F57Ae5b38EfCb70B796aA768A7e",
-        "operator_state_retriever_address": "0xd085215c0b4375B19946Dd87b14ef67Be35F9ff5",
+        "freeze_duration": "1209600s",
+        "finality_delay": "64",
+        "registry_coordinator_address": "0x70e0bA845a1A0F2DA3359C97E0285013525FFC49",
+        "operator_state_retriever_address": "0x8f86403A4DE0BB5791fa46B8e795C547942fE4Cf",
         "bls_apk_registry_address": "0xa7938Aa96355b45F293903f48fADDDDaeA030966",
-        "service_manager_address": "0xBf00805B2eB3eAE4aa0c0c9F9cD635357da53a90",
+        "service_manager_address": "0x998abeb3E57409262aE5b751f60747921B33613E",
```
Comment (alexanderbez) [inline delorean-1/genesis.json:146]: Just curious, these were changed recently? I just wanna make sure they're correct.

File: delorean-1/genesis.json [hunk 2]
```diff
@@ -371,15 +377,15 @@
             "amount": "10000000"
           }
         ],
-        "max_deposit_period": "3600s",
-        "voting_period": "3600s",
+        "max_deposit_period": "172800s",
+        "voting_period": "172800s",
```
Comment (alexanderbez) [inline delorean-1/genesis.json:381]: why did these change? Don't we want a small VP period?

File: delorean-1/genesis.json [hunk 3]
```diff
@@ -371,15 +377,15 @@
...
         "quorum": "0.334000000000000000",
         "threshold": "0.500000000000000000",
         "veto_threshold": "0.334000000000000000",
         "min_initial_deposit_ratio": "0.000000000000000000",
         "proposal_cancel_ratio": "0.500000000000000000",
         "proposal_cancel_dest": "",
-        "expedited_voting_period": "1800s",
+        "expedited_voting_period": "86400s",
```
Comment (alexanderbez) [inline delorean-1/genesis.json:388]: ditto

File: delorean-1/genesis.json [hunk 4]
```diff
@@ -481,23 +487,31 @@
       }
     },
     "params": null,
+    "poa": {
+      "params": {
+        "admins": [
+          "ethos10d07y265gmmuvt4z0w9aw880jnsr700j78lu59"
```
Comment (alexanderbez) [inline delorean-1/genesis.json:493]: What address is this? Shouldn't it be `ethos19w07e35lpuc6x4revruv29yv6jn3wzth3m5038` or did we change it?

File: delorean-1/genesis.json (hunk 1 above)
Comment (twoshark) [reply delorean-1/genesis.json:146]: I was curious about all these changes too, this is what I got from :
```
MONIKER=replace-me 
ethosd init $MONIKER --default-denom flux --chain-id delorean-1
```

Comment (alexanderbez) [reply delorean-1/genesis.json:146]: yeah these are defaults -- we spent time reviewing the genesis file and manually modified certain values that are specific to the testnet 👍 

I would undo these

File: delorean-1/genesis.json (hunk 4 above)
Comment (lucasege) [reply delorean-1/genesis.json:493]: this is a different admin, this is the PoA admin (for that module), however this is probably the default from that module, so yes we should change it to this `ethos19...` address imo

File: delorean-1/genesis.json [hunk 5]
```diff
@@ -481,23 +487,31 @@
...
         "template_client": {
           "chain_id": "",
           "trust_level": {
             "numerator": "1",
             "denominator": "3"
           },
-          "trusting_period": "2400s",
-          "unbonding_period": "3600s",
+          "trusting_period": "0s",
+          "unbonding_period": "0s",
```
Comment (lucasege) [inline delorean-1/genesis.json:514]:
```suggestion
          "trusting_period": "2400s",
          "unbonding_period": "3600s",
```

Comment (lucasege) [reply delorean-1/genesis.json:514]: i dont think we want to change these?

File: delorean-1/genesis.json [hunk 6]
```diff
@@ -592,7 +601,7 @@
     },
     "staking": {
       "params": {
-        "unbonding_time": "3600s",
+        "unbonding_time": "1814400s",
```
Comment (lucasege) [inline delorean-1/genesis.json:604]:
```suggestion
        "unbonding_time": "3600s",
```

File: delorean-1/genesis.json [hunk 7]
```diff
@@ -639,7 +648,7 @@
         "app": "0"
       },
       "abci": {
-        "vote_extensions_enable_height": "1"
+        "vote_extensions_enable_height": "0"
```
Comment (lucasege) [inline delorean-1/genesis.json:651]:
```suggestion
        "vote_extensions_enable_height": "1"
```

**Review by alexanderbez — APPROVED** [review]

**Pull request merged**